### PR TITLE
YD-410 Added white list properties

### PR DESCRIPTION
**Pull request by Bert-R**
There are now two properties, one whether invited users are checked against the white list and one for free sign up (users that are not invited).

### CODE REVIEW

File: core/src/main/java/nu/yona/server/subscriptions/service/UserService.java [hunk 1]
```diff
@@ -247,27 +252,33 @@ private void verifyUserDoesNotExist(String mobileNumber)
...
+	private void verifyUserIsAllowed(String mobileNumber, UserSignUp origin)
+	{
+		verifyBelowMaxUsers();
+		verifyWhiteList(mobileNumber, origin);
+	}
+
+	private void verifyBelowMaxUsers()
 	{
 		int maxUsers = yonaProperties.getMaxUsers();
-		if (maxUsers <= 0)
```
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/subscriptions/service/UserService.java]: This line was removed, which means `maxUsers` is always required.

File: core/src/main/resources/application.properties [hunk 2]
```diff
@@ -46,6 +46,8 @@ yona.supportedLocales=en-US,nl-NL
 yona.appleAppId = SFWNSYN8YK.com.xaton.yona
 # The maximum number of users accounts allowed. If set to negative or zero, it will fall back to white list mobile number verification.
```
Comment (aristotelos) [inline core/src/main/resources/application.properties]: This comment is no longer correct.

File: core/src/main/java/nu/yona/server/subscriptions/service/UserService.java (hunk 1 above)
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/subscriptions/service/UserService.java]: Indeed. There is no value in allowing negative values. If you don't like to use this property, just set it to a very large value (as we do today).

File: core/src/main/resources/application.properties (hunk 2 above)
Comment (Bert-R) [inline core/src/main/resources/application.properties]: Removed it.

**Pull request merged**